pkg/client/chains.go

```diff
@@ -5,6 +5,7 @@ import (
 )
 
 const (
+	mainnetURL = "https://tableland.network"
 	testnetURL = "https://testnet.tableland.network"
```

Comment:
Just want to be sure... we need to update the testnet url at some point in the future, correct? Or did we find a way to let the existing testnet url remain the same?

Reply:
See [here](https://www.notion.so/textile/Tableland-Testnet-Mainnet-separation-68c67911e68b410291cc03c911b97655#e647b9d032ed4c2b8d21f1976d03ebfe).

Reply:
Notice also what's described in that document regarding the testnet network.

Reply:
Ok so my interpretation of that is I should update this PR to use `testnets.tableland.network` for the `testnetURL`, correct?

Reply:
Yep